code-for-charlottesville/housinghub
# bootstrap integration (mostly

closes #60 

## Changes

- install bootstrap into the project
- migrate over main components to bootstrap layouts and components (note that I created https://github.com/code-for-charlottesville/housinghub/issues/66 to track this)

## How to test

- checkout the branch
- install latest dev dependencies `(cd frontend && npm install)`
- follow `frontend/README.md` file to get the app up and running"
- click around the app and make sure things look OK

Let me know if you have any questions!! happy to help 🌈 🎊
